code/modules/admin/topic.dm

```diff
@@ -1244,6 +1244,35 @@
 		else
 			show_traitor_panel(M)
 
+	else if(href_list["ickock"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/mob/M = locate(href_list["ickock"])
+		var/client/C = M.client
+		var/response = alert(usr, "Ick Ock!", "IC in OOC quick ban/note", "Note", "Ban", "Cancel")
+		if(response != "Cancel")
+			var/reason = input(usr, "Put the player's offending word(s) here.", "Why?", "")
+			if(response == "Note")
+				create_message("note", C.key, usr.ckey, "IC in OOC: " + reason, secret = 0, logged = 1, note_severity = "Minor", expiry = null)
+				usr.client.cmd_admin_pm(C.ckey, "IC in OOC is not allowed on this server, you have been noted for this incident.")

```

Comment:
Maybe "Please avoid talking about the current round in OOC. This can include fake IC in OOC as well as anything that could be interpreted as IC in OOC."?